### ISSUE TITLE
Is the VOC dataset's hyperparameter settings of config files the same as COCO dataset?

### ISSUE THREAD

**Issue by HommeLibre00** [Bohao-Lee/PDC#3]
Why when I try to reproduce the VOC dataset's experiment when using the same hyperparameter settings of COCO dataset, the result is much lower than you experiment?

**Comment by Bohao-Lee** [comment]
In the experiment, the learning rate of VOC differs from that of COCO. I'll update the config of VOC later.
